payments-core/src/main/java/com/stripe/android/model/parsers/DeferredPaymentIntentJsonParser.kt

```diff
@@ -22,7 +22,7 @@ class DeferredPaymentIntentJsonParser(
 
         val unactivatedPaymentMethods = jsonArrayToList(
             json.optJSONArray(FIELD_UNACTIVATED_PAYMENT_METHOD_TYPES)
-        )
+        ).map { it.lowercase() }
```

Comment:
OOC -- why does this need to be lowercase?

Reply:
Not sure. I copied what’s happening [here](https://github.com/stripe/stripe-android/blob/ffe8cf2f7ffca7a1358df589a84f7bbe936c97e4/payments-core/src/main/java/com/stripe/android/model/parsers/ElementsSessionJsonParser.kt#L35) in the `ElementsSessionJsonParser`.